### PR TITLE
Feat: Product page

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added product page

1: Filter
2: Product Cards

### CODE REVIEW

**Review by SanjayJatti — APPROVED** [review]
code is clean.

File: style/newarrival.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  height: 100%;
+}
+
+.newarrival-content{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.newarrival-title{
+    /* margin-bottom: 20px; */
```
Comment (amarphule) [inline style/newarrival.css]: remove comment

File: style/newarrival.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    font-weight: 100 !important;
+}
+
+@media (max-width: 767px) {
```
Comment (amarphule) [inline style/newarrival.css:32]: use single unit conversion either `px` or `rem` for your consistency

**Review by amarphule — CHANGES_REQUESTED** [review]
looks good to me. made some suggestions in the comments

**Comment by spraveenofficial** [comment]
> looks good to me. made some suggestions in the comments

Fixed Thanks amar

**Pull request merged**